packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3479,10 +3479,20 @@ export class ContainerRuntime
 		return result;
 	}
 
+	private stageControls: StageControls | undefined;
+	get inStagingMode(): boolean {
```

Comment:
Better to make explicit so later readers don't wonder.  Would recommend for the other unmarked methods around here too.
```suggestion
	public get inStagingMode(): boolean {
```